tests/app/views/contexts/conftest.py

```diff
@@ -207,3 +208,34 @@ def fake_session_data():
         response_id="response_id",
         case_id="case_id",
     )
+
+
+@pytest.fixture
+def summary_context_schema():
+    return load_schema_from_name("test_calculated_summary")
+
+
+@pytest.fixture
+def summary_context_answers():
```

Comment:
My preference would be these to be named after the schema itself. Not a blocker.